src/main/java/com/eva/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,5 @@
     public static final Prefix PREFIX_INDEX = new Prefix("i/");
     public static final Prefix PREFIX_LEAVE_START = new Prefix("ls/");
     public static final Prefix PREFIX_LEAVE_END = new Prefix("le/");
+    public static final Prefix PREFIX_INTERVIEW_DATE = new Prefix("id/");
```

Comment:
maybe instead of coming up with a myriad of prefixes when we make a new command like ls, le, id, we can standardize a date input to always just use PREFIX_DATE which is d: or d/ or something along these lines?